gdk/wizard/commands/data.py

```diff
@@ -0,0 +1,53 @@
+from gdk.common.configuration import get_configuration, _get_project_config_file
+import gdk.common.consts as consts
+import gdk.common.utils as utils
+import json
+
+
+class Wizard_data:
```

Comment:
Rename all the class names to camel case - `WizardData`

Reply:
Got it